test-it/ExpectedResults/MultiDimensionArrayDefaults.vtr

```diff
@@ -1,18 +1,12 @@
 (Line 4 Warning "Ignoring extra array initializer elements.")
 (Line 4 Warning "Ignoring extra array initializer elements.")
```

Comment:
Why are we printing the same warning twice? If you don't fix it, create a github issue and submit this.

Reply:
Good question. I'll create a github issue.